src/AppKit/NSBezierPath.cs

```diff
@@ -110,5 +110,15 @@ public unsafe void AppendPathWithGlyphs (uint[] glyphs, NSFont font)
 				_AppendPathWithGlyphs ((IntPtr)ptr, glyphs.Length, font);
 		}
 
+		public unsafe void AppendBezierPathWithCGGlyphs (uint[] glyphs, NSFont font)
```

Comment:
whats the difference with the previous method ? obviously they call a different API but I thought CGGlyphs were mapped to uint ? so it seems identical